test/testkit/matchers/log/log_http_matchers_test.go

```diff
@@ -28,24 +28,24 @@ var fls = []FlatLog{
 	},
 }
 
-var _ = Describe("HaveFlatLogs", func() {
+var _ = Describe("HaveFlatHTTPLogs", func() {
```

Comment:
Please use the same name pattern for everything related to the Flat(FluentBit|Otel)Logs